src/devices/bus/coco/cococart.cpp

```diff
@@ -141,6 +141,10 @@ cococart_slot_device::cococart_slot_device(const machine_config &mconfig, const
 
 void cococart_slot_device::device_start()
 {
+	m_cart_callback.resolve_safe();
+	m_nmi_callback.resolve_safe();
+	m_halt_callback.resolve_safe();
+
```

Comment:
Isn’t it a bad idea to call `resolve_safe` and then call `resolve` on the same delegates further down in the same function?

Reply:
Hmm. it appears I missed that. Fixed now.